pkg/cmd/plan.go

```diff
@@ -0,0 +1,212 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/ipfs/testground/pkg/api"
+	"github.com/ipfs/testground/pkg/config"
+
+	"github.com/BurntSushi/toml"
+	"github.com/go-git/go-git/v5"
+	gitcfg "github.com/go-git/go-git/v5/config"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/mattn/go-zglob"
+	"github.com/urfave/cli/v2"
+)
+
+var PlanCommand = cli.Command{
+	Name:  "plan",
+	Usage: "plan management",
+	Subcommands: cli.Commands{
+		&cli.Command{
+			Name:  "create",
+			Usage: "create plan `PLAN_NAME`",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:     "module",
+					Usage:    "Create module named `MODULE_NAME`",
+					Required: false,
+					Value:    "github.com/your/module/name",
+				},
+			},
+			Action: createCommand,
```

Comment:
This is cool. We'll have to refine the godocs a little to reflect how this works, but a command like this looks, on the surface, useful for the "quick start" user flow, as long as it doesn't introduce a lot of complexity.